Levi9.POS.Domain/Repositories/ClientRepository.cs

```diff
@@ -44,6 +44,29 @@ public async Task<bool> DoesClientExist(int clientId)
             var client = await _dbContext.Clients.FirstOrDefaultAsync(c => c.Id == clientId);
             return client != null;
         }
+        public bool CheckClientExist(int clientId)
+        {
+            return _dbContext.Clients.Any(c => c.Id == clientId);
+        }
+        public bool CheckEmailExist(string email)
+        {
+            return _dbContext.Clients.Any(c => c.Email == email);
+        }
+        public async Task<Client> UpdateClient(Client client)
```

Comment:
Namespace does not correspond to file location. It should be Levi9.POS.Domain.Repositories.